src/keymanager.ts

```diff
@@ -1,9 +1,37 @@
 import { PublicKey, PrivateKey, Checksum256, Signature } from '@greymass/eosio';
-import { GetKeyOptions, KeyManager, KeyManagerLevel, SignDataOptions, StoreKeyOptions } from 'tonomy-id-sdk';
+import { ES256KSigner, GetKeyOptions, KeyManager, KeyManagerLevel, SignDataOptions, StoreKeyOptions } from '@tonomy/tonomy-id-sdk';
 
 export default class JsKeyManager implements KeyManager {
-    signData(options: SignDataOptions): Promise<string | Signature> {
-        throw new Error('Method not implemented.');
+    async signData(options: SignDataOptions): Promise<string | Signature> {
+  
+        
+        const pv = localStorage.getItem('app.' + options.level);
```

Comment:
more verbose variable names is better. `privateKey`

Reply:
'app.' --> 'tonomy.id.app'

keeps it scoped and not conflict with other storage of apps